.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,28 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+    # will start the job at 13:15 UTC on 1st of every month
+    - cron: "15 13 1 * *"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Target branch to create requirements PR against"
+        required: true
+        default: 'main'
+
+jobs:
+
+  call-upgrade-python-requirements-workflow:
+     uses: edx/.github/.github/workflows/upgrade-python-requirements.yml@master
+     with:
+       branch: ${{ github.event.inputs.branch }}
+       team_reviewers: 'edx-data-engineering'
+       email_address: 'analytics@edx.opsgenie.net'
```

Comment:
I think we can use `data-engineering@edx.org` instead of `analytics@edx.opsgenie.net`

Reply:
Thanks for letting me know